new-findings.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
```

Comment:
Many of these commands could error out and should interrupt the script.  Add `set -e` to halt on first failure.  Wrap `make vet` commands with `set +e ; make ; set -e` to permit diagnostic reporting.

Reply:
Done.